### ISSUE TITLE
Destroying visible reveal does not remove is-reveal-open class from body.

### ISSUE THREAD

**Issue by GCheung55** [zurb/foundation-sites#10626]
#### How to reproduce this bug:

1. Click on the button to open the modal.

2. With JS, destroy the open reveal element.
```javascript
$('#exampleModal11').foundation('destroy');
```

3. Confirm modal is destroyed.
4. Check `body` element for `is-reveal-open` class.

#### What should happen:
Destroying the reveal element should remove the `is-reveal-open` class from the `body` element.

#### What happened instead:
`body` element retains `is-reveal-open` class after the reveal element is destroyed.

#### Browser(s) and Device(s) tested on:
Chrome 60.0.3112.113

#### Foundation Version(s) you are using:
6.4.3

#### Test case link:
https://codepen.io/anon/pen/WEqOrB - inside the modal is a link with a click event that destroys the reveal element. It will also print out a boolean determined by whether the `body` element has the `is-reveal-open` class or not.

Basically the `_destroy` method does not cleanup the `is-reveal-open` class, where as the `finishUp` method does.

**Comment by Arredatore** [comment]
Could it be linked to my topic? I also experience trouble with reveal https://github.com/zurb/foundation-sites/issues/10617

**Comment by iapassas** [comment]
You may consider the following 
    var alrt = new Foundation.Reveal($('#myReveal'));
    alrt.close();  
    if ($('body').hasClass('is-reveal-open')) { $('body').removeClass('is-reveal-open');  }
    var bl = $('.reveal-overlay').filter(function() {  return $(this).css('display') == 'block';  });
    $(bl).remove();

**Comment by drball** [comment]
I was having this problem and solved it by removing these animation effects: 
`data-animation-in="fade-in-fast" data-animation-out="fade-out"`
